apps/builder/app/builder/features/settings-panel/controls/url.tsx

```diff
@@ -476,7 +477,7 @@ export const UrlControl = ({
 
   const BaseControl = modes[mode].control;
 
-  const label = getLabel(meta, propName);
+  const label = humanizeString(getLabel(meta, propName));
```

Comment:
getLabel seem not needed anymore `humanizeString(meta.label ?? propName)`

Reply:
done